package.json

```diff
@@ -30,21 +30,21 @@
     "typed-error": "^2.0.0"
   },
   "devDependencies": {
-    "@types/bluebird": "^3.5.16",
-    "@types/body-parser": "^1.16.3",
+    "@types/bluebird": "3.5.16",
```

Comment:
These are changes I had locally to ensure we used fixed types versions because they break the build in patch version all the time